source/integrate/github-interoperability.rst

```diff
@@ -0,0 +1,188 @@
+
+Run the ``/github todo`` slash command to get a message with items to do in GitHub, including a list of unread messages and pull requests awaiting your review.
+
+Run the ``/github settings`` slash command to update your settings for notifications and daily reminders.
+
+Run the ``/github setup`` slash command to configure the integration between GitHub and Mattermost. This command has the following subcommands:
+
+- ``/github setup oauth``: Sets up the OAuth2 application in GitHub, establishing the necessary authorization connection between GitHub and Mattermost.
+- ``/github setup webhook``: Creates a webhook from GitHub to Mattermost, allowing real-time notifications and updates from GitHub to be sent to Mattermost channels.
+- ``/github setup announce``: Sends a message to designated channels in Mattermost, announcing the availability of the GitHub integration for team members to use.
+
+Frequently asked questions
+---------------------------
+
+How do I connect a repository instead of an organization?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set up your GitHub webhook from the repository instead of the organization. Notifications and subscriptions will then be sent only for repositories you create webhooks for. The reminder and ``/github todo`` searches the whole organization, but only show items assigned to you.
+
+How do I send notifications when a certain label is applied?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to send notifications to a Mattermost channel when **Severity/Critical** label is applied to any issue in the ``mattermost/mattermost-plugin-github`` repository, run the following slash command to subscribe to these notifications: ``/github subscriptions add mattermost/mattermost-plugin-github issues,label:"Severity/Critical"``
+
+How does the plugin save user data for each connected GitHub user?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GitHub user tokens are AES-encrypted with an **At Rest Encryption Key** configured in Mattermost. Once encrypted, the tokens are saved in the ``PluginKeyValueStore`` table in your Mattermost database.
+
+Get help
+--------
+
+Mattermost customers can open a `Mattermost support case <https://mattermost.zendesk.com/hc/en-us/requests/new>`_. To report a bug, please open a GitHub issue against the `Mattermost GitHub plugin repository <https://github.com/mattermost/mattermost-plugin-github>`_.
+
+For questions, feedback, and assistance, join our pubic `Integrations and Apps channel <https://community.mattermost.com/core/channels/integrations>`_ on the `Mattermost Community Server <https://community.mattermost.com/>`_ for assistance, or join us on the  `Mattermost Discussion Forum <https://forum.mattermost.org/c/plugins>`_.
```

Comment:
This discussion forum link is taking me to a "page doesn't exist" page in the forum